backend/src/types/user.types.ts

```diff
@@ -24,19 +28,22 @@ export const createUserSchema = z.object({
   googleId: z.string().min(1),
   profilePicture: z.string().optional(),
   bio: z.string().max(500).optional(),
-  hobbies: z.array(z.string()).default([]),
+  age: z.number().min(0).optional(),
+  location: z.string().optional(),
+  latitude: z.number().optional(),
+  longitude: z.number().optional(),
+  skillLevel: z.enum(SKILL_LEVELS as unknown as [string, ...string[]]).optional()
```

Comment:
I dont think we should cast `as unknown` here, it breaks type safety and makes the code harder to understand. 

We should do something like this to avoid type casting:

```suggestion
// change this in statics.ts
export const SKILL_LEVELS = ['Beginner', 'Intermediate', 'Expert'] as const;

skillLevel: z.enum(SKILL_LEVELS).optional(),
```